src/types.ts

```diff
@@ -1,3 +1,5 @@
+export type PlayerId = 0 | 1 | 2 | 3;
```

Comment:
For now I'll leave it as is. But once supporting manille for 6 players with some minor `manille` changes and some major `manille-demo` changes, I'll extend it to `0 | 1 | 2 | 3 | 4 | 5`